README.md

```diff
@@ -1,14 +1,12 @@
-pick(1)
-=======
+# Pick
 
-`pick(1)` allows users to choose one from set of choices using an
-`ncurses(3X)` interface with fuzzy search functionality.
+The `pick(1)` utility allows users to choose one option from a set of
+choices using an `ncurses(3X)` interface with fuzzy search functionality.
```

Comment:
`ncurses` has been replace with tputs and termios, #26. Might as well update that part too?